cilium/cilium.io
# feat: deps update 08 2023

**This PR:**
- Updates project dependencies.
- Updates tailwind config (As of Tailwind CSS v3.3 the line-clamp utilities are now included in the framework by default and this plugin is no longer required. [Link](https://github.com/tailwindlabs/tailwindcss-line-clamp))
- Updates mobile adaptation "NetworkPolicy Editor" blog post. [Original](https://cilium.io/blog/2021/02/10/network-policy-editor/) [After update](https://deploy-preview-338--cilium.netlify.app/blog/2021/02/10/network-policy-editor/);
- Adds YoutubeIframe component;

**Steps to test**

- Open the [All pages](https://deploy-preview-338--cilium.netlify.app/)
- Confirm that everything looks and works as expected
